mureq.py

```diff
@@ -36,51 +65,69 @@ def request(method, url, *, read_limit=None, **kwargs):
     :return: Response object
     :rtype: Response
     :raises: HTTPException
-    """
+    """  # noqa: E501
     with yield_response(method, url, **kwargs) as response:
         try:
             body = response.read(read_limit)
         except HTTPException:
             raise
-        except IOError as e:
+        except OSError as e:
             raise HTTPException(str(e)) from e
-        return Response(response.url, response.status, _prepare_incoming_headers(response.headers), body)
+        return Response(
+            response.url,
+            response.status,
+            _prepare_incoming_headers(response.headers),
+            body,
+        )
 
 
-def get(url, **kwargs):
-    """get performs an HTTP GET request."""
-    return request('GET', url=url, **kwargs)
+def get(url: str, **kwargs) -> "Response":
+    """Get performs an HTTP GET request."""
+    return request("GET", url=url, **kwargs)
 
 
-def post(url, body=None, **kwargs):
-    """post performs an HTTP POST request."""
-    return request('POST', url=url, body=body, **kwargs)
+def post(url: str, body: bytes | None = None, **kwargs) -> "Response":
+    """Post performs an HTTP POST request."""
+    return request("POST", url=url, body=body, **kwargs)
 
 
-def head(url, **kwargs):
-    """head performs an HTTP HEAD request."""
-    return request('HEAD', url=url, **kwargs)
+def head(url: str, **kwargs) -> "Response":
+    """Head performs an HTTP HEAD request."""
+    return request("HEAD", url=url, **kwargs)
 
 
-def put(url, body=None, **kwargs):
-    """put performs an HTTP PUT request."""
-    return request('PUT', url=url, body=body, **kwargs)
+def put(url: str, body: bytes | None = None, **kwargs) -> "Response":
+    """Put performs an HTTP PUT request."""
```

Comment:
Is it really correct to capitalize the method name in the docstring?

Reply:
I removed the method name from the comment strings and added a kwargs note in latest commit